OpenEdXMobile/src/main/java/org/edx/mobile/social/SocialSourceType.kt

```diff
@@ -0,0 +1,21 @@
+package org.edx.mobile.social
+
+enum class SocialSourceType(
+    private val values: List<String>
+) {
+    UNKNOWN(listOf("unknown")),
```

Comment:
Can we update the type to `EMAIL`, as we have only these possible ways to sign-in/registeration?

Reply:
Email is handled separately. Plus it's not related to Social Auth.